### DISCUSSION TITLE
Figma files / UI kit mentioned on the website but not in source or resources

### DISCUSSION THREAD

**Discussion by morekitsch** [wonderflow-bv/wanda#287] [General]
Totally dumb question, but the website mentioned Figma files and I simply can't find them. Not on the Wanda websites 'resources' section or within the source code.  Am I missing something?

**Comment by design-wonderflow** [comment]
No dumb question, currently only Themes are shared (https://www.figma.com/file/kcYVgrOqcGIiD2jBjpP3MV/01-%E2%80%94-Themes?node-id=2%3A2). We will fix the documentation accordingly asap. Thanks for asking

**Comment by dmerello-wip** [comment]
Wanda is great! 
it would be really usefull a figma file to work with designers!
